wpl_cli/src/group/instructions/asset/data.rs

```diff
@@ -0,0 +1,12 @@
+use clap::Parser;
+use solana_sdk::pubkey::Pubkey;
+
+#[derive(Debug, Parser, Clone)]
+pub struct AssetArgs {
+    /// Collection mint
+    #[arg(short = 'm', long, value_parser = clap::value_parser!(Pubkey))]
+    pub mint: Pubkey,
```

Comment:
Is collection=group in this context? if so, this `mint` could be renamed to something more obvious to the end user

Reply:
The reason why "mint" is specified here is because we have a group PDA as well, so when that's required we could refer it as "group/collection" and when the mint is needed, this command would help, as anyways the argument comes around a subcommand of `group update --mint` or `asset royalty add --mint`, hence I felt it should be self-explanatory.